### ISSUE TITLE
Suggestion switch to writexl or openxls [R]

### ISSUE THREAD

**Issue by grantmcdermott** [chaisemartinPackages/did_multiplegt_dyn#78]
I'm not sure that I understand the necessity to write Excel files (rather than CSVs or even .Rdata files). But given the pain that folks have had with `xlsx`, might it not make sense to switch to a lighter option like [writexl](https://cran.r-project.org/web/packages/writexl/index.html) or [openxlsx](https://cran.r-project.org/web/packages/openxlsx/index.html) instead? Both of these drop the dreaded Java dependency.

(Note: These two packages are both focused on _writing_ Excel files. If you need to read them back, then you can combine with [readxl](https://readxl.tidyverse.org/index.html), which is also very lightweight.)

**Comment by lbiagini75** [comment]
I agree with you. Indeed, I don't save in .xlsx but in a text. After that, I can easily edit the output with an R code to create a LaTeX table.
I use this code:

```
  # Save results
  sink(results_name.txt)
  print(summary(model_dCDH24))
  sink()
```

**Comment by jcvdav** [comment]
Are there any updates on this? I'm trying to run code on a server and reducing the number of dependencies would go a long way in helping things go smoothly.

**Comment by DiegoCiccia** [comment]
Hello,
This is Diego from the team.
A new version of DIDmultiplegtDYN (v.2.1.0) was pushed yesterday to Github. In this version, we drop the xlsx dependecy in favor of openxlsx. You can download the source file [here](https://github.com/chaisemartinPackages/did_multiplegt_dyn/blob/main/dist/DIDmultiplegtDYN_2.1.0.tar.gz) or you can install it directly from Github using the code snippet in the [home page](https://github.com/chaisemartinPackages/did_multiplegt_dyn/tree/main?tab=readme-ov-file#github-development-version). 
We are finishing the revision of the package and we will soon commit the updated version on CRAN.
I hope this helps!
Best,
Diego

**Comment by grantmcdermott** [comment]
Thanks @DiegoCiccia.

You probably know this, but in case not: You can automatically close linked issues as part of a PR by using [keywords](https://github.blog/news-insights/product-news/closing-issues-via-commit-messages/) in your commit message or PR description, e.g. `Closes #78`, `Fixes #78`, etc.
